lib/config/presets/internal/monorepo.ts

```diff
@@ -287,6 +287,7 @@ const repoGroups = {
   gitbeaker: 'https://github.com/jdalrymple/gitbeaker',
   'github-workflows-kt':
     'https://github.com/typesafegithub/github-workflows-kt',
+  'gocloud.dev': 'https://github.com/google/go-cloud',
```

Comment:
```suggestion
  'gocloud-dev': 'https://github.com/google/go-cloud',
```
we should avoid dots

Reply:
How come?

We could otherwise drop the `.dev` altogether. Just thought it made sense as that's the name of the package (see https://github.com/google/go-cloud/blob/master/doc.go#L48).

Alternatively we could just use the name of the repository instead of the name of the package i.e. `go-cloud`.

Reply:
`go-cloud` seems good

Reply:
Done